internal/k8s/validation.go

```diff
@@ -499,7 +499,7 @@ func validateAppProtectDosOnlyAnnotation(context *annotationValidationContext) f
 func validateAppProtectDosLogDestAnnotation(context *annotationValidationContext) field.ErrorList {
 	allErrs := field.ErrorList{}
 
-	err := api_validation.ValidateAppProtectLogDestination(context.value)
+	err := api_validation.ValidateAppProtectDosAccessLogDest(context.value)
```

Comment:
The `ValidateAppProtectLogDestination` was for `app-protect-dos-security-log-destination` and `ValidateAppProtectDosAccessLogDest` was for `app-protect-dos-access-log-destination
Now u joined them together so I think change the name for the function for example to `ValidateAppProtectDosLogDest` will be more correct

Reply:
fixed